components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/ConfigurationFacade.java

```diff
@@ -90,7 +91,7 @@ public ExternalIdPConfig getIdPConfigByName(String idpName, String tenantDomain)
                     log.debug("A registered IdP was not found the given name");
                 }
             }
-        } catch (IdentityApplicationManagementException e) {
+        } catch (IdentityProviderManagementException e) {
             log.error("Exception while getting IdP by name", e);
```

Comment:
Ideally we must be throwing this exception and its the client who can decide what to do with it. Lets change this to throw the exception and in authentication framework we can catch the exception, log it and continue.